tests/_support/index.js

```diff
@@ -735,39 +735,24 @@ Cypress.Commands.add("clusterCleanup", function(fn) {
 });
 
 Cypress.Commands.add("visitUrl", function(options) {
-  var callback = function() {};
-
-  if (options.logIn && options.remoteLogIn) {
-    callback = function(win) {
-      // {"uid":"ui-bot","description":"UI Automated Test Bot","is_remote":true}
-      win.document.cookie =
-        "dcos-acs-info-cookie=" +
-        "eyJ1aWQiOiJ1aS1ib3QiLCJkZXNjcmlwdGlvbiI6IlVJIEF1dG9tYXRlZCBUZXN0IEJvdCIsImlzX3JlbW90ZSI6dHJ1ZX0K";
-    };
-  } else {
-    callback = function(win) {
-      // {"uid":"ui-bot","description":"UI Automated Test Bot"}
+  cy.visit(Cypress.env("CLUSTER_URL") + "/#" + options.url, {
+    onBeforeLoad(win) {
       win.document.cookie =
-        "dcos-acs-info-cookie=" +
-        "eyJ1aWQiOiJ1aS1ib3QiLCJkZXNjcmlwdGlvbiI6IlVJIEF1dG9tYXRlZCBUZXN0IEJvdCJ9Cg==";
-    };
-  }
+        options.logIn && options.remoteLogIn
+          ? // {"uid":"ui-bot","description":"UI Automated Test Bot","is_remote":true}
+            "dcos-acs-info-cookie=eyJ1aWQiOiJ1aS1ib3QiLCJkZXNjcmlwdGlvbiI6IlVJIEF1dG9tYXRlZCBUZXN0IEJvdCIsImlzX3JlbW90ZSI6dHJ1ZX0K"
+          : // {"uid":"ui-bot","description":"UI Automated Test Bot"}
+            "dcos-acs-info-cookie=eyJ1aWQiOiJ1aS1ib3QiLCJkZXNjcmlwdGlvbiI6IlVJIEF1dG9tYXRlZCBUZXN0IEJvdCJ9Cg==";
 
-  if (options.identify && options.fakeAnalytics) {
-    var identifyCallback = callback;
-    callback = function(win) {
-      identifyCallback(win);
+      // mock a global analytics object
       win.analytics = {
```

Comment:
Where and how do we use that global object?

Reply:
i believe that the usage is limited to the tracking plugin

Reply:
thanks for asking this! 

in a next iteration we might only add that stuff to window, if the tracking plugin is enabled. i would prefer to not add this to the scope of this PR though.